samples/apps/copilot-chat-app/webapp/src/components/chat/ChatHistoryItem.tsx

```diff
@@ -166,7 +173,35 @@ export const ChatHistoryItem: React.FC<ChatHistoryItemProps> = ({ message, getRe
                 <div className={isMe ? mergeClasses(classes.item, classes.me) : classes.item}>
                     <div className={classes.header}>
                         {!isMe && <Text weight="semibold">{fullName}</Text>}
-                        <Text className={mergeClasses(classes.time, classes.alignEnd)}>{time}</Text>
+                        <Text className={classes.time}>{time}</Text>
+                        {isBot &&
+                            <Dialog>
```

Comment:
nit: I would move this to its own separate component i.e., PromptDetails to keep components modularized